### ISSUE TITLE
Incorrect event for ServerEvents.tags?

### ISSUE THREAD

**Issue by AJBuilder** [xMGZx/UnifyTags#26]
I'm getting this error when running the 1.17 script on 1.19.2 Minecraft.

```
[System] [CHAT] 1) unify.js#370: Error occurred while handling event 'ServerEvents.tags': TypeError: Cannot find function replaceInput in object dev.latvian.mods.kubejs.server.tag.PreTagEventJS@46844d32.
[System] [CHAT] More info in 'logs/kubejs/server.log'
```

Looking more at the script I noticed this:

```
if (global["V6P"]) {
    ServerEvents.tags("item", e_recipes)
    ServerEvents.recipes(e_recipes)
} else {
    onEvent("tags.items", e_tags_items)
    onEvent("recipes", e_recipes)
}
```

e_recipes looks to be modifying/unifying recipes, why is it being called when the server is working on tags?

Changing to:

```
if (global["V6P"]) {
    ServerEvents.tags("item", e_tags_items)
    ServerEvents.recipes(e_recipes)
} else {
    onEvent("tags.items", e_tags_items)
    onEvent("recipes", e_recipes)
}
```

fixed the issue.
The weird part is the script was working fine for a while, and randomly stopped working.

**Comment by mathgeniuszach** [comment]
That is very strange how the script was working for a while despite that... that's definitely a typo on my end.

**Comment by mathgeniuszach** [comment]
Fixed in 27af24d7b841c164acb8e2fac4dda3c54fa99240, thanks for reporting!
